src/Microsoft.ServiceFabric.Actors/Runtime/ActorReminder.cs

```diff
@@ -124,6 +124,14 @@ internal void ArmTimer(TimeSpan newDueTime)
             {
                 try
                 {
+                    this.actorManager.TraceSource.WriteInfoWithId(
+                        TraceType,
+                        this.actorManager.GetActorTraceId(this.OwnerActorId),
+                        "Arming timer for reminder ({0}), actor: ({1}), newDueTime: ({2})",
+                        this.Name,
+                        this.OwnerActorId.GetStorageKey(),

```

Comment:
Why storage key? Let us ensure we trace actor id every where not confuse with storage keys in someplaces.

Reply:
Storage key is just {Actor Type}_{Actor Id}, this might help us keep track of the actor id type for the particular actor.